### DISCUSSION TITLE
Getting "Failed to create the chia daemon" depending on CA

### DISCUSSION THREAD

**Discussion by maxpla3** [Chia-Network/chia-blockchain#10600] [Support]
I have setup a new node in my LAN. When I copy its CA to the harvester and do chia init -c ca_dir, after trying to start harvester I'm getting "Failed to create the chia daemon". No errors in the log. If I switch to the CA of the old node, there is no issue. All chia software is 1.2.11.

**Comment by esaung** [comment]
Sounds like a bad CA file, try deleting the CA file and try creating a new one

**Comment by thkaw** [comment]
I encouted similar problem.
The CA confirmed OK and every config properly.
Even I type **chia stop all** but looks some reamins process make this problem happens...

The solution is reboot haverster the try to start chia harvester.

**Reply by maxpla3** [reply]
I found a chia process still running in background even after chia stop all. I had to kill that process first. After that everything worked fine.

**Reply by loppefaaret** [reply]
you need `chia stop all -d` for it to kill the daemon process as well
